src/runtime/registry/matomo-analytics.ts

```diff
@@ -4,10 +4,12 @@ import { boolean, object, optional, string } from '#nuxt-scripts-validator'
 import type { RegistryScriptInput } from '#nuxt-scripts'
 
 export const MatomoAnalyticsOptions = object({
-  matomoUrl: string(), // site is required
-  siteId: string(),
+  matomoUrl: optional(string()),
```

Comment:
hm maybe we can deprecate this in favour if `trackerUrl`

Reply:
probably, because in the official documentation this name is used as an environment variable - apparently as syntax sugar

```js
var u="//{$MATOMO_URL}/";
    _paq.push(['setTrackerUrl', u+'matomo.php']);
```

https://developer.matomo.org/guides/tracking-javascript-guide